src/examples/button/SplitButtonSizes.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { SplitButton, Button, ChevronButton } from '@zendeskgarden/react-buttons';
+import { Dropdown, Trigger, Menu, Item } from '@zendeskgarden/react-dropdowns';
+
+const Example = () => (
+  <Row>
```

Comment:
Add `alignItems="center"` for consistency with other [size](https://5ef13c3303616c01cc6490d3--zendeskgarden.netlify.app/components/button#sizes) examples.